evm/src/cpu/kernel/interpreter.rs

```diff
@@ -1163,9 +1273,11 @@ impl<'a> Interpreter<'a> {
             self.generation_state.observe_contract(tip_h256)?;
         }
 
-        if self.halt_offsets.contains(&offset) {
-            self.running = false;
+        if !self.is_kernel() {
+            self.add_jumpdest_offset(offset);
+            log::debug!("Added {offset} to jdt")
```

Comment:
nit: We're already printing the final jumpdest table I think? So this may be redundant info that we can skip?
If we keep it, then maybe move it to `trace` and expand `jdt` for clarity

Reply:
Yes. I forgot to delete this.